HW1/P6/P6.txt

```diff
@@ -1 +1,22 @@
 # Your discussion here
+Phrases: 
+
+Phrase 0 borne up. have led my rag-of-muffins where they should invite them without cause. [Kneels] Now all the world- why he's
+
+Phrase 1 offer into envy. Ye turn me to leave Is only bitter to your former malady, That should be cozened. Fare
+
+Phrase 2 cittern-head. The head of thine eye but with light skirmishes enfeebled. [Here they bear it? No. 'Tis insensible then? Yea,
+
+Phrase 3 than yours? Write them together, yours is worth a limit past my praise, master? In bodily health, sir. Marry, the
+
+Phrase 4 about! Arise, and say it is true indeed. So your face were but a filthy vice; say to thyself 'From
+
+Phrase 5 Cressida. If beauty have a care of yours my sails Must fill, or else To him that in this trunk
+
+Phrase 6 and feet together. Sea-water shalt thou exchange for rags?- robes, for tittles?- titles, for thyself? -me. Thus expecting thy reply,
+
+Phrase 7 thankless natures- abhorred spirits!- Not all these hideous fears, And madly play with me, For can raise no money bid
+
+Phrase 8 sorrow say. Exeunt 2. Before PAGE'S house Enter and at it, Roaring for Troilus; who hath so far forth as
+
+Phrase 9 They break their hearts from them, And tell me We have always truly serv'd you; and shall not stay; For
```

Comment:
In general, nice coding style, and clear commenting. Even though many of your algorithms took different methods as mine, because of your commenting, there were very easy to follow. Good job!